docs/src/features/utilities.md

```diff
@@ -7,6 +7,16 @@ in
 TangentSpace
 ````
 
+# Fallback for the exponential map: Solving the corresponding ODE
+
+When additionally loading [`NLSolve.jl`](https://github.com/JuliaNLSolvers/NLsolve.jl) the following fallback for the exponential map is available.
```

Comment:
NLSolve is for approximate inverse retraction, not the exponential map.

Reply:
Hm? I am confused. It is either approximating Log or exp but approximating a retraction would not make sense (also I think I did not change this, but to fix errors is of course always good).

Reply:
NLSolve is for approximating Log, for Exp we need OrdinaryDiffEq.

Reply:
That (to me) also sounds a little strange to have two dependencies there, but then ist log probably, yes.